src/Microsoft.OData.Edm/Csdl/Parsing/Common/EdmXmlDocumentParser.cs

```diff
@@ -407,40 +407,64 @@ protected string ValidateEnumMembersPath(string path)
 
         private string ValidateTypeName(string name)

```

Comment:
The syntax of method and the method name looks mismatch. If it's only to validate the type name, the return type 'string' doesn't make sense?

Reply:
@xuzhg I'm not sure I understand... This private method existed with that signature. Are you suggesting that I should rename it?